### PR TITLE
Email template for APD registration

### PR DESCRIPTION
**Pull request by WitchellJack**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@(params: Map[String, Any])
+
+HMRC received your Air Passenger Duty registration
```
Comment (elabeca) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.txt]: Please remove this - this is the subject

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "HMRC received your Air Passenger Duty registration"){
+    <h1>You submitted a registration online</h1>
```
Comment (elabeca) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.html]: Please remove this H1

Comment (elabeca) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.html]: Also remove the text inside it "You submitted a registration online"

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "HMRC received your Air Passenger Duty registration"){
```
Comment (elabeca) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.html]: Please replace "HMRC received your Air Passenger Duty registration" with "You submitted a registration online"

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.txt (hunk 1 above)
Comment (WitchellJack) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.txt]: so would you like me to remove both lines: 
@(params: Map[String, Any])

and

@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "HMRC received your Air Passenger Duty registration"){

**Pull request merged**